### DISCUSSION TITLE
All the marvel snap Cards & Locations .ZIP (Current)

### DISCUSSION THREAD

**Discussion by paperc07** [AdriaGual/marvel-snap-bot#71] [General]
Here is all the currents cards in a zip file if anyone can do anything with them they also all came in .webp extension but I also provided a link if you want the jpg version
[Marvel Snap Cards](https://github.com/AdriaGual/marvel-snap-bot/files/10708455/Marvel.Snap.Cards.zip)
[Marvel Snap Cards (JPG)](https://github.com/AdriaGual/marvel-snap-bot/files/10708538/Marvel.Snap.Cards.JPG.zip)
[Marvel Snap Locations](https://github.com/AdriaGual/marvel-snap-bot/files/10759680/locations.zip)

**Comment by PeanutJar-Jelly** [comment]
I tried using some of these cards just to see and it throws an error that looks like it's due to the image dimensions of these.  I haven't tried to crop them but I suspect they don't work since they aren't the same resolution as the bot expects.

ValueError: needle dimension(s) exceed the haystack image or region dimensions

**Reply by paperc07** [reply]
Yeah I figured it would someone previously said that was an issue. So with that being said do we know the dimensions of the screen shotted cards I wonder if we could just make them the same. Also I just more or less put this out just incase in the future someone can do something with these images 🤷‍♂️

**Reply by ZloyPupsic** [reply]
140x155 is a good ratio

**Reply by ZloyPupsic** [reply]
for now images won't work, because you need screenshot of a card in game in several hand positions: left, center, right

**Reply by paperc07** [reply]
Yeah that's what they said.😔
